lib/fastlane/plugin/dynatrace/helper/symlink_helper.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module Fastlane
+  module Helper
+    class SymlinkHelper
+      def self.path_exists?(path)
+        unless path.nil?
+          return Dir.exist?(path) || File.exist?(path)
+        end
+        return false
+      end
+
+      def self.symlink_lldb(lldb_path, destination_path)
+        require_path(destination_path)
+        require_path(lldb_path)
+        symlink(lldb_path, destination_path)
+      end
+
+      def self.auto_symlink_lldb(destination_path)
```

Comment:
I'd add some logging so it's traceable what's happening here.

Reply:
done ✅